app/src/main/java/org/traccar/client/Position.java

```diff
@@ -139,4 +140,7 @@ public void setBattery(double battery) {
         this.battery = battery;
     }
 
+    private boolean mock;
+
+    public boolean getMock() { return mock; }
```

Comment:
Setter is missing.

Reply:
Done

Reply:
It's still not fixed.

Reply:
Which line please ?

Reply:
You need a line break after open bracket and before closing bracket.

Reply:
Well I tried to stick to the existing style, hope it's ok now